afp.conf

```diff
@@ -0,0 +1,6 @@
+[Global]
+log file = /var/log/netatalk.log
+
+[share]
+    path = /share
```

Comment:
COPY: Why change away from `/media` or why not use a directory like `/media/share`. This kinda matches the FHS 3.0 as /media contains mount points for removable media which it kinda will be with the described way of mounting an external directory into the container.